test/lit.cfg.py

```diff
@@ -67,6 +67,8 @@
 
 if config.enable_bindings_python:
   llvm_config.with_environment('PYTHONPATH', [
-      os.path.join(config.torch_mlir_python_packages_dir, 'torch_mlir'),
-  ],
-                               append_path=True)
+      os.pathsep.join([
+          os.path.join(config.torch_mlir_python_packages_dir, 'torch_mlir'),
+          os.environ['PYTHONPATH'],
```

Comment:
why is this needed?

Reply:
I think this was required to make it work with our local environment and was accidentally included. Will revert